test/e2e/pkg/deploy/controller.go

```diff
@@ -64,13 +64,3 @@ func (w *Deployment) DeployWorkspacesController() error {
 
 	return nil
 }
-
-func (w *Deployment) CustomResourceDefinitions() error {
```

Comment:
Is it really removed from Makefile https://github.com/amisevsk/devworkspace-operator/blob/add-conversion-v1alpha2/Makefile ?
Or it just does not make sense to execute it separately since e2e tests do make install, which includes crds installing as well?